frontend/src/components/pages/VerifyEmail.tsx

```diff
@@ -44,8 +44,8 @@ const VerifyEmailContent = ({
       return (
         <>
           <Text variant="display-md" paddingBottom="1vw">
-            Please verify your account from the
-            link we sent to your email address!
+            Please verify your account from the link we sent to your email
+            address!
```

Comment:
Ahhh this is my bad, I tried to just make some easy changes without running the linter, so this will cause some merge conflicts